pkg/collector/providers/file.go

```diff
@@ -295,5 +296,10 @@ func GetCheckConfigFromFile(name, fpath string) (check.Config, error) {
 		config.LogsConfig = rawLogsConfig
 	}
 
+	// DockerImages entry was found: we ignore it if no ADIdentifiers has been found
+	if cf.DockerImages != nil && cf.ADIdentifiers == nil {
+		return config, errors.New("the 'docker_images:' section is deprecated, please use 'ad_identifiers:' instead")
```

Comment:
I'd remove the `:` at the end of the section names for readability

Reply:
👍